scripts/serve_hs.py

```
@@ -0,0 +1,313 @@
+        ),
+    )
+    parser.add_argument(
+        "--port",
+        type=int,
+        required=True,
+        help="Port to listen on.",
+    )
+    parser.add_argument(
+        "--host",
+        type=str,
+        default="0.0.0.0",
```

Ruff (0.16.1)

[error] Possible binding to all interfaces

(S104)